reana_workflow_controller/consumer.py

```diff
@@ -154,6 +156,15 @@ def _update_workflow_status(workflow, status, logs):
             try:
                 workflow_engine_logs = _get_workflow_engine_pod_logs(workflow)
                 workflow.logs += workflow_engine_logs + "\n"
+                dask_client = dd.Client("tcp://dask-scheduler.default.svc.cluster.local:8786")
```

Comment:
svc address will be different, and can be controller in DaskCluster configuration:
```
apiVersion: kubernetes.dask.org/v1
kind: DaskCluster
metadata:
  name: dask # set to e. g. workflow ID
```